en/research-summaries/business-account-research-summary.html

```diff
@@ -1,185 +1,6 @@
----
-layout: void
```

Comment:
Can you check if the void layout is used elsewhere? If not we might as well remove it with this request.

Reply:
I just want to focus on Research Summaries here.